### PR TITLE
raise error if CVE bug missing labels

### PR DESCRIPTION
**Pull request by Ximinhan**
if a bug missing tracker label bug have CVE-XXXX in summary, raise the suspicious bug

### CODE REVIEW

**Review by joepvd — CHANGES_REQUESTED** [review]
I think the test failures of previous iterations actually tell something. In this implementation, it is assumed that there is a summary, even if that field does not exist. This should either get initialized in `Bug().__init__`, or it should first be tested if the property exists.

Could you add a test for when this code path hits?

Also, could you rebase this into a single commit, and open the PR from your fork rather than from the openshift repository?

**Review by thiagoalessio — CHANGES_REQUESTED** [review]
+1 in Joep's change requests.
Also, can we have more descriptive commit messages, and a corresponding ART JIRA (feel free to create one describing the bug in case it doesn't exist yet) ?

File: elliottlib/bzutil.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def whiteboard_component(self):
         raise NotImplementedError
 
     def is_tracker_bug(self):
+        if not set(constants.TRACKER_BUG_KEYWORDS).issubset(set(self.keywords)) and re.search(r'^CVE-(\d+)-(\d+)', self.summary):
```
Comment (thegreyd) [inline elliottlib/bzutil.py:61]: @Ximinhan Let's not have this logic in this method. I propose to have it in a separate function, 
```python
def looks_like_tracker_bug(self):
    return re.search(r'^CVE-(\d+)-(\d+)', self.summary)
```

**Comment by thegreyd** [comment]
@Ximinhan We should think about where to raise this type of warning / issue. One place could be check-bugs job, and IMO this fits nicely there than say find-bugs:sweep. For that we would need to have this in verify-bugs and verify-attached-bugs command

**Pull request closed without merging**